packages/shared/src/components/search/SearchBarSuggestion.tsx

```diff
@@ -73,16 +74,13 @@ export const SearchBarSuggestion = ({
 
   return (
     <Button
-      spanClassName="w-fit my-2 flex-shrink tablet:line-clamp-1"
-      textPosition="justify-start"
       icon={<AiIcon />}
-      buttonSize={ButtonSize.Medium}
-      className={classNames(
-        'btn-secondary !h-auto min-h-[2.5rem] w-fit border-theme-divider-tertiary text-theme-label-tertiary typo-subhead',
-        className,
-      )}
+      variant={ButtonVariant.Subtle}
+      className={classNames('w-fit !justify-start typo-subhead', className)}
       onClick={handleSuggestionsClick}
       {...props}
-    />
+    >
+      <span className="w-fit tablet:!line-clamp-1">{children}</span>
```

Comment:
Tested existing implementations if the ellipsis still works as intended.

https://preview.app.daily.dev/search/chat

![Screenshot 2024-02-19 at 9 35 31 PM](https://github.com/dailydotdev/apps/assets/13744167/c89f326d-1a17-41a2-aa69-29cff6093bf9)